packages/react/src/components/SafeAnchor/index.js

```diff
@@ -77,6 +77,7 @@ const SafeAnchor = React.forwardRef((props, ref) => {
       'aria-disabled': true,
     };
   }
+  // eslint-disable-next-line react-hooks/rules-of-hooks
   if (innerRef) mergeRefs = useMergedRefs(ref, innerRef);
```

Comment:
TODO:
```
const mergeRefs = useMergedRefs(ref, innerRef || null);
```
Since useMergedRefs supports null ref.
https://github.com/react-restart/hooks/blob/master/src/useMergedRefs.ts#L13

Reply:
Thanks, it's fixed.